src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -76,6 +76,13 @@ A state container approach familiar to many web developers.
   by Marcelo Glasberg
 * [Flutter meets Redux: The Redux way of managing Flutter applications state][],
   by Amir Ghezelbash
+  
+## Fish-Redux
+
+Fish Redux is an assembled flutter application framework based on Redux state management. It is suitable for building medium and large applications.
+
+* [Fish-Redux-Library][], by Alibaba
+* [Flutter-Movie][] - Large and complex example on how to develop using Fish Redux
```

Comment:
"Large and complex" doesn't sound like a compliment.  :)

=>
[Flutter movie app][]: Non-trivial example demonstrating how to use Fish Redux.